recipes/toolbox_utils/meta.yaml

```
@@ -0,0 +1,74 @@
+{% set name = "toolbox_utils" %}
+{% set version = "2.0.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/toolbox_utils-{{ version }}.tar.gz
+  sha256: ad447a48aec23b70238bf6f89bbadcb33441df75bd7e4f75881072705758e7f5
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - dateparser
+    - tabulate
+    - numpy
+    - scipy
+    - pandas
+    - pint-pandas
+    - pint
+    - pydantic
+    - setuptools
+    - setuptools_scm
+    - typing-extensions
+    - odfpy >=1.4.1
+    - xlrd >=2.0.1
+    - openpyxl >=3.0.7
+    - pyxlsb >=1.0.8
+    - pytables >=3.6.1
+    - matplotlib >=3.3.2
+    - python
+    - pip
```

I don't think most of these are needed at build-time:
```suggestion
    - setuptools
    - setuptools_scm
    - python >=3.8
    - pip
```

---

Done in recent commit...